batchflow/opensets/cifar.py

```diff
@@ -71,9 +71,15 @@ def _gather_extracted(all_res):
             test_data = _gather_extracted(all_res)
         logger.info("Extracted")
 
+        images = np.concatenate([train_data[0], test_data[0]])
+        labels = np.concatenate([train_data[1], test_data[1]])
+
         self._train_index = DatasetIndex(np.arange(len(train_data[0])))
-        self._test_index = DatasetIndex(np.arange(len(test_data[0])))
-        return train_data, test_data
+        self._test_index = DatasetIndex(np.arange(len(train_data[0]), len(train_data[0]) + len(test_data[0])))
```

Comment:
Giving meaningful names like `test_len`, `train_len`, `total_len` would both increase readability and is reusable